dev/Repeater/ViewManager.cpp

```diff
@@ -668,8 +689,12 @@ winrt::UIElement ViewManager::GetElementFromElementFactory(int index)
     }
 
     repeater->AnimationManager().OnElementPrepared(element);
-    repeater->OnElementPrepared(element, index);
-    m_phaser.PhaseElement(element, virtInfo);
+
+    if (!itemsSourceContainsElements)
```

Comment:
Is it possible to reorder this method so that there is only on if(!itemsSourceContainsElements) block?

Reply:
no, the order matters here.